sound/soc/codecs/rt5682.c

```diff
@@ -2726,6 +2725,7 @@ static void rt5682_calibrate(struct rt5682_priv *rt5682)
 	mutex_lock(&rt5682->calibrate_mutex);
 
 	rt5682_reset(rt5682);
+	regmap_write(rt5682->regmap, RT5682_I2C_CTRL, 0x000f);
```

Comment:
is it safe or useful to do this in SoundWire mode?

Reply:
It is safe and it is not related with SoundWire mode.

Reply:
ok, thanks. Applied the changes directly